File: app/src/main/java/com/kksionek/gdzietentramwaj/view/AdProviderInterface.java
package com.kksionek.gdzietentramwaj.view;

import android.content.Context;
import android.location.Location;
import android.view.ViewGroup;

public interface AdProviderInterface {
    void showAd(ViewGroup adView);
    void resume();
    void pause();
    void loadAd(Context context, Location location);
}

File: app/src/main/java/com/kksionek/gdzietentramwaj/view/AdProviderInterface.kt
package com.kksionek.gdzietentramwaj.view

import android.content.Context
import android.location.Location
import android.view.ViewGroup

interface AdProviderInterface {
    fun initialize(context: Context, adMobAppId: String)
    fun showAd(adView: ViewGroup)
    fun resume()
    fun pause()
    fun loadAd(context: Context, location: Location)
}
